app/src/main/res/layout/book_description.xml

```diff
@@ -0,0 +1,145 @@
+            android:layout_height="wrap_content"
+            app:layout_constraintTop_toBottomOf="@id/title_element"
+            android:text="@string/author"
+            android:textAlignment="center"
+            android:textSize="18sp"
+            android:textColor="@color/white"
+            android:fontFamily="@font/poppins" />
+
+        <ImageView
+            android:id="@+id/book_image"
+            android:layout_width="150dp"
+            android:layout_height="225dp"
+            app:layout_constraintTop_toBottomOf="@id/author_element"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            android:background="@drawable/artemis_fowl_cover"/>
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
+            android:layout_marginVertical="8dp"
+            android:background="@color/white"
+            android:layout_marginTop="16dp"
+            app:layout_constraintTop_toBottomOf="@+id/book_image" />
+
+        <TextView
+            android:id="@+id/rating_header"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginVertical="8dp"
+            android:text="@string/rating"
+            app:layout_constraintTop_toBottomOf="@id/book_image"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            android:textSize="18sp"
+            android:textColor="@color/white"
+            android:fontFamily="@font/poppins"
+            />
+
+
+        <RatingBar
+            android:id="@+id/rating_bar_element"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            style="@style/ratingBarSmall"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/rating_header"
+            android:isIndicator="true"
+            android:numStars="5"
+            android:progressTint="@color/white"
+            android:rating="3"
+            android:stepSize="0.1"
+            />
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
+            android:layout_marginVertical="8dp"
+            android:background="@color/white"
+            android:layout_marginTop="16dp"
+            app:layout_constraintTop_toBottomOf="@+id/rating_bar_element" />
+
+
+        <TextView
+            android:id="@+id/book_details_heading"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:layout_constraintTop_toBottomOf="@id/rating_bar_element"
+            android:fontFamily="@font/poppins"
+            android:text="@string/details"
+            android:textColor="@color/white" />
```

Comment:
I noticed when running this branch that the bottom dividing line was intersecting this text element, A top and right margin should make it look better! Also text size 18sp differentiates it from the body text :)
```suggestion
            android:textColor="@color/white" 
            android:layout_marginStart="8dp"
            android:layout_marginTop="16dp"
            android:textSize="18sp" />
```